pages/api/Schema/reports/designationAccounts/designationAccounts.graphql

```diff
@@ -1,10 +1,30 @@
 extend type Query {
-  designationAccounts(accountListId: ID!): [DesignationAccount!]!
+  designationAccounts(accountListId: ID!): [DesignationAccountsGroup!]!
+}
+
+extend type Query {
+  activeDesignationAccount(
```

Comment:
Usually mutations have some sort of verb or otherwise indicate what the action is in their name.
```suggestion
  setActiveDesignationAccount(
```